notifier.rst

```diff
@@ -16,7 +16,7 @@ the users (e.g. SMS, Slack messages, emails, push notifications, etc.). The
 Notifier component in Symfony is an abstraction on top of all these
 channels. It provides a dynamic way to manage how the messages are sent.
 Get the Notifier installed using:
-
+uid
```

Comment:
Is it really needed?

Reply:
Ups ... fix on the way, thanks!

Reply:
I think it was already removed by Javier

Reply:
Yes, removed while merging 👍